apps/services/auth/admin-api/src/app/v2/delegations/test/delegation-admin.auth.spec.ts

```
@@ -132,4 +134,89 @@
+    beforeEach(async () => {
+      app = await setupAppWithoutAuth({
+        AppModule,
+        SequelizeConfigService,
+        dbType: 'postgres',
+        beforeServerStart: async (app) => {
+          await new Promise((resolve) =>
+            resolve(
+              app.use(
+                bodyParser.json({
+                  verify: (req: any, res, buf) => {
+                    if (buf && buf.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)